cardano/base/templatetags/tags.py

```diff
@@ -13,3 +13,16 @@ def lookup_country_vouched(countries, key):
 def lookup_country_total(countries, key):
     q = max(countries, key=lambda d: (d['name'] == key))
     return q['total_diff']
+
+
+@register.filter
```

Comment:
Also you could instead of using ``safe`` filter again and again just mark it as safe string in the template.

Note: Abusing ``safe`` in django is not a good idea when there are different ways to implement something.